debian/control

```diff
@@ -6,7 +6,8 @@ Standards-Version: 4.3.0.3
 Build-Depends:
     debhelper-compat (= 11),
     dh-python,
-    python3-all,
+    python3 (>= 3.6),
+    python3-pytest,
     python-gobject-2,
```

Comment:
Should this be replaced or removed? I think this is deprecated.

Reply:
`autopkgtest` failed with `python3-all`, but also without a python interpreter, this version is the one I got to run. `python3-pytest` is needed because apparently the tests are run during the `debuild -S -sa` process.

Reply:
BTW I think I can cut down on the dependencies listed here, both `Build-Depends` and `Depends`, I'll go through them one at a time. But first, integration tests.

Reply:
Sorry, I meant `python-gobject-2`. Isn't that deprecated and for Python 2?